include/KokkosInterface.h

```diff
@@ -15,15 +15,17 @@ namespace sierra {
 namespace nalu {
 
 using HostSpace = Kokkos::HostSpace;
-using DeviceSpace = Kokkos::DefaultExecutionSpace;
+//!!!! Important: the 'Kokkos::Serial' here makes sure we only run serial. !!!!
+// using DeviceSpace = Kokkos::DefaultExecutionSpace;
+using DeviceSpace = Kokkos::Serial;
```

Comment:
OK. Thanks... I'll check that and make sure something else in Nalu doesn't break.